clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py

```diff
@@ -173,7 +173,8 @@ def write_new_cassandra_config(self, seeds_list, destructive_restart=False):
             run_command("chown -R cassandra /var/lib/cassandra")
 
         self.start_cassandra()
-        os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
+        if os.path.exists("/etc/clearwater/force_cassandra_yaml_refresh"):
+            os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
```

Comment:
It's more pythonic (and generally better) to catch the OSError and ignore it if `e.errno == errno.ENOENT`.

https://docs.python.org/2/glossary.html#term-eafp